### DISCUSSION TITLE
Cognative differences between save states in new projects beta

### DISCUSSION THREAD

**Discussion by oberocks** [community/community#8686] [Projects and Issues]
First off, I quite like and appreciate the new Projects (beta) option, and find myself enjoying the ability to switch between the card and spreadsheet/grid view for longer lists - or simply a better macro scan of issues *across* repos for a brand/firm! 

So 🙇 for the extra vision the new digs provide!!! 🚀 

One thing that tripped me up for only yesterday... and honestly this is one of those lazy user scenarios where I didn't read the entire menu... but basically it took me a while to figure out the green dot relative to the UI drop down menu trigger in a Project (beta) grid view *tab*, is actually a signal to remember to press save on the drop down menu link that's there to actually save the changes.

I refreshed and went back and fourth before realizing my error of the state save process - and honestly figured it's either intended to reset each time, or its my own user error, or a bug. But it didn't feel like a bug... so I just let it be and went to bed.

Now, my laziness aside, I did realize just today... where my cognitive gap came from! 

> As a still clueless first time user, the main title and tab name changes are saved to a server upon a blur (or maybe on key down/up too - didn't check yet). The title edit-ability was especially noticeable in the flow of how my first beta project got created. 
> 
> However, if i add new columns to a grid view - in that case I need to trigger the drop down menu and click the correct item/link to save the changes with a quick sequence of interactions. Different from some of the other editable data points, but no big deal once I know how to save changes.
> 
> And just like that, it's even more awesome now! 😄

**TL;DR** I just wanted to flag this one, as maybe a UXer somewhere might be looking for some design forks in the road to test for... and in my own experience, with admittedly lazy behaviors during my EOD and random test drive of the feature, a series of needed design choices might have accidentally impacted my overall impression of the work in a ever so slightly negative way... but then again... such a thing gave me a learning buzz once I figured out where I was not picking up the hint already designed in there to lead me to the bowl. So maybe this is one of those micro interaction flows to keep feelers on.

Cheers and again, thank you for the thoughtful and appreciated work!
